src/main/java/seedu/address/logic/commands/DeleteNotesCommand.java

```diff
@@ -0,0 +1,87 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Notes;
+import seedu.address.model.person.Person;
+
+/**
+ * Deletes Notes from the person identified using it's displayed index from the address book.
+ */
+public class DeleteNotesCommand extends Command {
+
+    public static final String COMMAND_WORD = "delete_notes";
```

Comment:
might want to adhere to the new command format of no _

Reply:
Good catch!

Reply:
That's a good point but i was planning to merge it first than change it with the other command formats.

Reply:
I think ill change it here and just make this PR close both issues